app/helpers/success_alert_helper.rb

```diff
@@ -12,4 +12,11 @@ def access_and_permissions_granted_params(application_id, granting_access:, user
       }
     end
   end
+
+  def access_removed_params(application_id, user: current_user)
+    {
+      message: "Access removed",
+      description: access_removed_message(application_id, user),
+    }
+  end
```

Comment:
I'm not sure I see the distinction between the two methods in this file. The first one includes `granted` in the name but it would also apply if permissions have been removed (or even some granted, some removed), so I'm not sure the granted/removed distinction holds?

Could the method added in the previous commit maybe be rejigged here to accommodate the three options (e.g. adding `removing_access:` to the method signature), or a clearer distinction made? This would extend the conditional logic in the helper but allow the view to stay 'simple'